CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ All notable changes to TTT2 will be documented here. Inspired by [keep a changel
 
 ### Added
 
+- Added hook ENTITY:ClientUse(), which is triggered clientside if an entity is used
+  - Return true if only clientside should be used
```

Comment:
I'm not sure I like that. Is there a problem if ClientUse is called and after that also Use on the server? Because if you did not add any code to the Use function on the server, it wouldn't matter anyway

Reply:
This only stops it from being sent to the server. We just omit a net message like that.
As it is a hook, you generally dont call it yourself. You just write code clientside that is executed on use. If you do not return true, after its called clientside, its automatically called serverside. Its meant to prevent server usage in UI only cases.

Reply:
Hmm, I think I'm ok with that then. But this definitely needs a second opinion, so I'll leave this thread open so the other reviewer cag give their opinion on that as well